python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -786,6 +792,82 @@ def maskSaturatedPixels(self, badAmpDict, ccdExposure, detector, detectorConfig,
                 badAmpDict[ampName] = True
                 maskView |= maskView.getPlaneBitMask("BAD")
 
+        # Edge bleed masking
+        if self.config.doITLEdgeBleedMask and detector.getPhysicalType() == 'ITL':
+
+            # Get footprint of saturated pixels
+            thresh = afwDetection.Threshold(limits[self.config.saturatedMaskName])
+            fpList = afwDetection.FootprintSet(maskedImage, thresh).getFootprints()
+
+            # Go through footprints to select saturated cores
+            for i in range(len(fpList)):
+                if fpList[i].getArea() > 10000.:
+
+                    # Get centroid of saturated core
+                    xCore, yCore = fpList[i].getCentroid()
+                    # Get the number of saturated columns around the centroid
+                    widthSat = numpy.sum(fpList[i].getSpans().asArray(), axis=1)[int(yCore)]
+                    for amp in ccdExposure.getDetector():
+                        # Select 2 amplifiers around the saturated
+                        # core with a potential edge bleed
+                        if amp.getBBox().contains(xCore, yCore) \
+                                or amp.getBBox().contains(xCore, numpy.abs(4000.-yCore)):
+
+                            # Get the saturation level in that amp
+                            ampName = amp.getName()
+                            satLevel = metadata[f"LSST ISR SATURATION LEVEL {ampName}"]
+
+                            if ampName[:2] == 'C1':
+                                # Check there is an edge bleed
+                                if numpy.mean(maskedImage.image.array[:50, int(xCore)-5:int(xCore)+5]) \
+                                        > 0.8*satLevel:
```

Comment:
Where did 0.8 come from?  Is this empirical?  I'm not loving this because right now we define satLevel as the PTC turnoff, but we may increase it later depending.  Does this need to be smarter or configurable?